frontend/components/users/profile/ProfilePhotoPanel.tsx

```
@@ -1,166 +1,67 @@
-        />
-        <div className="absolute bottom-0 inset-x-0">
-          <div className="flex items-center justify-center bg-black bg-opacity-50 text-white text-lg font-semibold py-2">
-            Edit
+    <>
+      <div className="flex items-center border border-core-outline p-6 rounded-lg space-x-4">
+        <div
+          className="relative h-32 w-32 lg:h-44 lg:w-44 rounded-full overflow-hidden cursor-pointer group"
+          onMouseEnter={() => setIsHovered(true)}
+          onMouseLeave={() => setIsHovered(false)}
+          onClick={() => setIsModalOpen(true)}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)